### PR TITLE
Retrieve and set suggested reviewer field to user context struct

### PR DESCRIPTION
**Pull request by kimlisa**
part of https://github.com/gravitational/teleport/issues/4937

#### Description
Retrieve `suggested reviewers` list and add it in user context struct

### CODE REVIEW

File: lib/web/ui/usercontext.go [hunk 1]
```diff
@@ -39,6 +39,14 @@ type accessStrategy struct {
 	Prompt string `json:"prompt"`
 }
 
+// AccessCapabilities defines allowable access request rules defined in a users roles.
```
Comment (a-palchikov) [inline lib/web/ui/usercontext.go:42]:
```suggestion
// AccessCapabilities defines allowable access request rules defined in the user's roles.
```

**Review by a-palchikov — APPROVED** [review]

**Review by r0mant — APPROVED** [review]

**Review by russjones — APPROVED** [review]
Bot.

File: lib/web/ui/usercontext.go [hunk 2]
```diff
@@ -83,8 +91,8 @@ type UserContext struct {
 	Cluster *Cluster `json:"cluster"`
 	// AccessStrategy describes how a user should access teleport resources.
 	AccessStrategy accessStrategy `json:"accessStrategy"`
-	// RequestableRoles are roles that the user can assume when requesting access.
-	RequestableRoles []string `json:"requestableRoles"`
+	// AccessCapabilities defines allowable access request rules defined in a users roles.
```
Comment (webvictim) [inline lib/web/ui/usercontext.go:94]:
```suggestion
	// AccessCapabilities defines allowable access request rules defined in a user's roles.
```

**Comment by russjones** [comment]
@kimlisa Since `branch/v6` is split off, can this be merged?

File: lib/web/ui/usercontext.go (hunk 2 above)
Comment (kimlisa) [reply lib/web/ui/usercontext.go]: revised

**Review by fspmarshall — APPROVED** [review]

**Pull request merged**